docs/src/modules/components/overview/mainDemo/PickerButton.tsx

```diff
@@ -32,13 +32,17 @@ function ButtonDateField(props: DatePickerFieldProps) {
       variant="outlined"
       size="small"
       startIcon={<CalendarTodayRoundedIcon fontSize="small" />}
-      sx={{ minWidth: 'fit-content' }}
+      sx={[
+        { minWidth: 'fit-content' },
+        ...(Array.isArray(pickerContext.rootSx) ? pickerContext.rootSx : [pickerContext.rootSx]),
+      ]}
```

Comment:
Why is SX an array here? 🤔

Reply:
`sx` can be an array when you want to apply multiple ones
Here is the type of `sx`:

```ts
export type SxProps<Theme extends object = {}> =
  | SystemStyleObject<Theme>
  | ((theme: Theme) => SystemStyleObject<Theme>)
  | ReadonlyArray<
      boolean | SystemStyleObject<Theme> | ((theme: Theme) => SystemStyleObject<Theme>)
    >;
```

I think the goal is to allow to merge `sx` declarations without caring about property override and let `@mui/system` take care of it.

Reply:
Ah, cool, didn't know about that 👍